README.md

```diff
@@ -178,13 +178,18 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@master
-      - uses: actions/setup-ruby@v1
-        with:
-          ruby-version: '2.6'
+      steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - run: |
+          git fetch --prune --unshallow
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+      - name: Setup pronto
       - run: gem install pronto pronto-rubocop
-      - run: PRONTO_PULL_REQUEST_ID="$(jq --raw-output .number "$GITHUB_EVENT_PATH")" PRONTO_GITHUB_ACCESS_TOKEN="${{ secrets.GITHUB_TOKEN }}" pronto run -f github_status github_pr -c origin/master
+      - run: PRONTO_PULL_REQUEST_ID="$(jq --raw-output .number "$GITHUB_EVENT_PATH")" PRONTO_GITHUB_ACCESS_TOKEN="${{ github.token }}" bundle exec pronto run -f github_status github_pr -c origin/${{ github.base_ref }}
```

Comment:
I also has to add `.pull_request.number` vs `.number` not sure if you experienced this as well?

Reply:
seems working fine for me :-
![image](https://user-images.githubusercontent.com/16918568/81668908-fa950f00-9462-11ea-8e72-3116ed647e06.png)

Reply:
When adding `bundle exec` the following error occurs `Bundler::GemNotFound`. I don't believe the Workflow creates a `Gemfile`, causing this error. Perhaps this should be removed?

Reply:
yes, you are right but since I have used with Gemfile and bundler is now part of the lastest ruby.
Running command directly was causing ```Pronto not found`` issue. I am thinking to write a wiki for specific integrations where such things can be mentioned.

Reply:
Yes, but for this README, to @jaydorsey 's comment below, most people will copy and paste this snippet. In order for `bundle exec` to work, there should be the steps of `bundle install` etc before hand. If not, simply removing `bundle exec` allows for the minimum requirements needed to get this working as an isolated event.

Reply:
@smridge updated, please review.

Reply:
Nice Wiki! 👍